gossipd/gossip.c

```diff
@@ -1485,7 +1485,7 @@ static void setup_listeners(struct daemon *daemon, u16 portnum)
 	/* IPv6, since on Linux that (usually) binds to IPv4 too. */
 	fd1 = make_listen_fd(AF_INET6, &addr6, sizeof(addr6), true);
 	if (fd1 >= 0) {
-		struct sockaddr_in6 in6;
+		struct sockaddr_in6 in6 = {};
 
 		len = sizeof(in6);
 		if (getsockname(fd1, (void *)&in6, &len) != 0) {
```

Comment:
Does not `getsockname` either initialize `in6` or we do not use `in6` at all?

Reply:
See previous comment :-)